gbpservice/nfp/scripts/mode_shift.sh.enterprise

```diff
@@ -0,0 +1,233 @@
+#! /bin/bash
+
```

Comment:
We don't need to submit this file. This needs to be fixed for latest code. So, lets review this as next step.